testdata/project-v3/PROJECT

```diff
@@ -3,19 +3,45 @@ layout: go.kubebuilder.io/v3-alpha
 projectName: project-v3
 repo: sigs.k8s.io/kubebuilder/testdata/project-v3
 resources:
-- crdVersion: v1
+- api:
+    namespaced: true
+    version: v1
+  controller: true
+  domain: testproject.org
   group: crew
   kind: Captain
   version: v1
-  webhookVersion: v1
-- crdVersion: v1
+  webhooks:
+    defaulting: true
+    validating: true
+    version: v1
```

Comment:
we need keep the webhook version here.

Reply:
Next push will include:
- `resources[].api.version` -> `resources[].api.crdVersion`
- `resources[].webhooks.version` -> `resources[].webhooks.webhookVersion`